recipes/blackdoc/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "blackdoc" %}
+{% set version = "0.1.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1573230228009268b506ea2590ecedc41c8604b86706645b177bd7d64193db19
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - blackdoc=blackdoc.__main__:main
+  script: SETUPTOOLS_SCM_PRETEND_VERSION={{ version }} {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools >=45
```

Comment:
You don't need setuptools since you have pip

Reply:
true, but `pip` doesn't limit the version of `setuptools`. I'm not quite sure why I chose that version, but the source's `pyproject.toml` specifies `setuptools>=42` so the recipe should copy that.